hack/e2e-kind-cluster-setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -xe
+
+CNI_VERSION=${CNI_VERSION:-0.4.0}
+OCI_BIN=${OCI_BIN:-docker}
+IMG_REGISTRY=${IMAGE_REGISTRY:-localhost:5000/maiqueb}
+
+setup_cluster() {
+    git clone https://github.com/k8snetworkplumbingwg/multus-cni/
+    trap "popd && cleanup" EXIT
+    pushd multus-cni/e2e
```

Comment:
instead pushd / popd you can use, if desired
(but then maybe need set -E as well, so errors in subshell will be caught IIRC)

```
(
...
)
```

Reply:
I'd rather improve the readability; I am not a bash person, and the less tricks I see / use the better ...

Thanks for the suggestion though.